ilastik/applets/counting/countingGui.py

```diff
@@ -647,10 +651,19 @@ def setupLayers(self):
 
         inputDataSlot = self.topLevelOperatorView.InputImages
         if inputDataSlot.ready():
-            inputLayer = self.createStandardLayerFromSlot( inputDataSlot )
-            inputLayer.name = "Input Data"
-            inputLayer.visible = True
-            inputLayer.opacity = 1.0
+            inputLayer = None
+            for i in range(len(layers)):
+                if layers[i].name == "Raw Input":
+                    inputLayer = layers[i]
+                    layers[i], layers[-1] = layers[-1], layers[i]
```

Comment:
Hey, for things like these, where you swap stuff, it would be really helpful to add a line of comment on why (not obvious to me)